src/main/java/io/vertx/mqtt/MqttServerOptions.java

```diff
@@ -252,10 +252,29 @@ public MqttServerOptions setTimeoutOnConnect(int timeoutOnConnect) {
     return this;
   }
 
+  /**
+   * Do the same thing as {@link MqttClientOptions#setKeepAliveTimeSeconds(int)}. Use it instead.
+   */
+  @Deprecated
+  @Override
+  public MqttServerOptions setIdleTimeout(int idleTimeout) {
+    super.setIdleTimeout(0);
+    return setTimeoutOnConnect(idleTimeout);
+  }
+
+
   /**
    * @return  timeout on CONNECT before closing connection
    */
   public int timeoutOnConnect() {
     return this.timeoutOnConnect;
   }
+
+  /**
+   * @return  the TCP idle timeout, in seconds,In MqttServer the Parameter should be 0, it will use timeoutOnConnect.
+   */
+  @Override
+  public int getIdleTimeout() {
+    return 0;
```

Comment:
Why returning 0 here? The method will return zero by default.

And, thinking about it... I think we can also delegate the method call to `MqttServerOptions#timeoutOnConnect`